traitcuration/static/styles/layout/layout.scss

```diff
@@ -2,3 +2,4 @@
 
 @import "./footer/footer.scss";
 @import "./header/header.scss";
+@import "./sidebar/sidebar"
```

Comment:
Is there a reason why footer and header are imported using their filenames directly (with `.scss` extension), but sidebar in some different way? I'm not an expert in Sass, so the reason is probably there and I just don't know it

Reply:
Actually no, the files with `.scss` extension were part of the scss folder structure template I am using and the sidebar was imported manually, that's the cause of the difference. I will define a follow up action to do it one way across all imports to maintain consistency

Reply:
OK! In this case, could you either fix it in this PR, or add a follow-up action in existing ticket https://github.com/EBIvariation/trait-curation/issues/40, or create a new ticket for this action? Just so that it doesn't get lost in the discussions

Reply:
Yes, I am adding that in this PR

Reply:
Great, this means that this action is the only one remaining for this PR. Once you address it, feel free to merge this PR without further review from my side